pkg/tsdb/v3iotsdb.go

```diff
@@ -182,31 +172,34 @@ func (a *V3ioAdapter) Querier(_ context.Context, mint, maxt int64) (*querier.V3i
 	return querier.NewV3ioQuerier(a.container, a.logger, mint, maxt, a.cfg, a.partitionMngr), nil
 }
 
-func (a *V3ioAdapter) DeleteDB(config bool, force bool) error {
-
-	path, filter := a.partitionMngr.GetHead().GetTablePath()
-	a.logger.Info("Delete partition %s", path)
-	err := utils.DeleteTable(a.container, path, filter, a.cfg.QryWorkers)
-	if err != nil && !force {
-		return err
+func (a *V3ioAdapter) DeleteDB(configExists bool, force bool, fromTime int64, toTime int64) error {
```

Comment:
IMHO, it is better to provide an input struct with default values instead of separate params.